be/src/Project/Services/Identity/Identity.Api/Areas/Accounts/Resources/AccountResources.de.resx

```diff
@@ -156,6 +159,12 @@
   <data name="LastNameRequiredErrorMessage" xml:space="preserve">
     <value>Geben Sie Ihren Nachnamen ein</value>
   </data>
+  <data name="MarketingApprovalHeader" xml:space="preserve">
+    <value>Marketingformular für Unternehmer, die mit ELTAP zusammenarbeiten.</value>
+  </data>
+  <data name="MarketingApprovalText" xml:space="preserve">
+    <value>Für den Fall, dass Sie Informationen über unser aktuelles Angebot, laufende Kampagnen, implementierte Funktionen, Änderungen sowie andere von ELTAP durchgeführte Maßnahmen erhalten möchten, bitten wir um das Ausfüllen der folgenden Marketing-Einwilligungen. Die erteilten Einwilligungen können jederzeit widerrufen werden. Der Widerruf hat jedoch keine Auswirkungen auf bereits vor dem Widerrufsdatum getroffene Maßnahmen.</value>
```

Comment:
**suggestion:** Word ELTAP couldn't be in text, because this app is not used only by ELTAP